arch/arm/src/xmc4/xmc4_lowputc.c

```diff
@@ -109,6 +109,138 @@
 
 #define UART_OVERSAMPLING    16
 
+#define IS_POWER_OF_TWO(x) (((x) != 0) && (((x) & ((x) - 1)) == 0))
```

Comment:
let's reuse the macro from: https://github.com/apache/nuttx/pull/12427/commits/596ed0ebfec2dcaf5e88712a207c05e88bf4d988

Reply:
Oh Nice ! I was actually also looking for a Macro for the log2 to void my tbctr_size and rbctr_size helper fonctions !

Reply:
So waiting for : https://github.com/apache/nuttx/pull/12427 to be merged